### DISCUSSION TITLE
Ports

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#4677] [Support] [answered]
Hello, What ports does the panel use?

**Comment by parkervcp** [comment, marked as answer]
The panel is a standard web application and runs on 80 and 443 behind your web server. By default Wings runs on 2022 and 8080. 

Beyond that the ports needed depend on the servers you want to run.

**Reply by ghost** [reply]
Okay, Do I need to add 3306 and 6379.

**Reply by parkervcp** [reply]
Only if you want to have access to the db or redis outside the server. Those are generally internal only.

**Reply by ghost** [reply]
Okay, Thanks For The Help!

**Reply by kavionyxstudio** [reply]
> Okay, Do I need to add 3306 and 6379.

You can also add in your 50-server.cnf from your MariaDB in `/etc/mysql/mariadb-conf/50-server.cnf` put the IP on 0.0.0.0 instead of 127.0.0.1 and giving yourself the full access to the database. Then you can reach it from the outside.
